Makefile

```diff
@@ -5,11 +5,16 @@
 #
 
 TARGET = kata-proxy
+SOURCES := $(shell find . 2>&1 | grep -E '.*\.go$$')
 
-all: $(TARGET)
+VERSION_FILE := ./VERSION
+VERSION := $(shell grep -v ^\# $(VERSION_FILE))
+COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
```

Comment:
I think this COMMIT_NO is too long. Cloud we cut it into 12 bytes?

Reply:
I guess we could. We could also cut it down to 7 bytes using `git rev-parse --short HEAD`. But this value is only logged once so isn't going to bloat the logs. As such, I'm not really sure why we need to truncate it?

Reply:
Yes, I'm fine with full SHA as well.